pennylane_forest/device.py

```diff
@@ -178,9 +178,9 @@ class ForestDevice(QubitDevice):
             variable ``QUILC_URL``, or in the ``~/.forest_config`` configuration file.
             Default value is ``"http://127.0.0.1:6000"``.
     """
-    pennylane_requires = ">=0.6"
+    pennylane_requires = ">=0.8"
     version = __version__
-    author = "Josh Izaac"
+    author = "Rigetti Computing Inc."
```

Comment:
I don't know exactly who all worked on this, but I don't think it's unfair to give @josh146 some of the credit!

Reply:
This is part of the information that shows up when you query the device information in PennyLane. I thought best to sync it with the author field in `setup.py` :)

I think this field hasn't changed since I first prototyped this plugin, so definitely worth updating!

Reply:
I think it'd definitely be unfair not to give @josh146 credit! It means we should probably update `setup.py` as well. I would've been fine just to stick with the original author (i.e. Josh), but if the view is to include everyone who's authored the library since, maybe something like "Rigetti, Xanadu" or even "Rigetti, Xanadu and friends" (since we now have contributions from others as well)?